### DISCUSSION TITLE
VSCode not requesting authorization for github when signing into copilot

### DISCUSSION THREAD

**Discussion by SergeantSar** [community/community#53479] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When I install the github copilot extension on vscode, it installs and tells me that "This extension is enabled globally," however, I am not prompted to sign into my github account for authorization even though every source tells me that I should be and copilot is non-functional. The icon for copilot that should be in the bottom right is also not present. I have tried looking into multiple solutions, including making everything up to date, completely uninstalling vscode via revo uninstaller, to no avail.

Any help would be much appreciated.

**Comment by Bibooo25730** [comment]
I tried to install github copilot and vscode told me to authorize and brought up this page https://github.com/github-copilot/signup?editor=copilot_1.83.41
I'm not sure if this will help you

**Reply by SergeantSar** [reply]
Unfortunately not, only redirects to my copilot settings.
Somehow the copilot extension is just refusing to even check that I'm verified and I'm not sure why.
